custom/conf/app.example.ini

```diff
@@ -2484,3 +2484,18 @@ ROUTER = console
 ;PROXY_URL =
 ;; Comma separated list of host names requiring proxy. Glob patterns (*) are accepted; use ** to match all hosts.
 ;PROXY_HOSTS =
+
+; [actions]
+;; Enable/Disable actions capabilities
+;ENABLED = false
+;DEFAULT_ACTIONS_URL = https://gitea.com
```

Comment:
What is this URL for? Are there cases where the app url differs from the actions url?

Reply:
This indicates where should we find the relative path action plugin. i.e. when use a action like
```yaml
name: versions

on:
  push:
    branches:
      - main
      - releases/*

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
```

Now we need to know how to get `actions/checkout`, this configuration is the default git server to get it. That means we will get the repository via `git clone ${DEFAULT_ACTIONS_URL}/actions/checkout` and fetch tag `v3`.

To help people who don't want to mirror these actions in their git instances, the default value is `https://gitea.com`
To help people run actions totally in their network, they can change the value and copy all necessary action repositories into their git server.

Reply:
mh, ok. Thank you for explaining this to me 🙂 looking forward to using it